mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -46,13 +46,22 @@ examples:
     vars:
       display_name: 'my-query'
       dataset_id: 'my_dataset'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'bigquerydatatransfer_config_cmek'
+    skip_test: true
```

Comment:
Any particular reason to skip the test? If so it looks like the added field is not covered by any test.

Reply:
This was based off of another example from `bigquery_dataset` which includes cmek support but doesn't include a test: https://github.com/GoogleCloudPlatform/magic-modules/pull/2226

I believe we skip it due to the test including a cryptoKeyRing creation, which we are unable to remove: https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings

I've included a test that includes the `kmsKeyName` field.